File: final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
  int finalValueAfterOperations(vector<string>& operations) {
    int result = 0;

    for ( auto value : operations ) {
      if ( value[1] == '+' ) {
        result++;
      } else {
        result--;
      }
    }

    return result;
  }
};

File: final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function(operations) {
  let cnt = 0

  for ( let i = 0; i < operations.length; i++ ) {
    if ( operations[i][1] === '-' ) {
      cnt--
    } else {
      cnt++
    }
  }

  return cnt
};